src/app/services/loaders/objects/phoenix-objects.ts

```diff
@@ -97,8 +97,8 @@ export class PhoenixObjects {
     // Jet energy parameter can either be 'energy' or 'et'
     let length = (jetParams.energy ? jetParams.energy : jetParams.et) * 0.2;
     // Ugh - We don't want the Jets to go out of the event display
-    if (length > 2000) {
-      length = 2000;
+    if (length > 3000) {
```

Comment:
Fine for now, but I think this is something we should make configurable (the appropriate maximum length will depend a lot on the experiment)

Reply:
I agree. May be we can apply some sort of normalization.

It's also possible to check this directly in the loaders and change the energies from there.